### PR TITLE
Make README more friendly to new users

### LINKED ISSUE #138: webdriverjs vs selenium-wedriverjs (question)
Hello everyone,

I apologize in advance if I'm asking stupid questions.

First, this project is in fact different from [selenium-wedriver-js](https://code.google.com/p/selenium/wiki/WebDriverJs) is that correct? The name threw me off for a little while :(

Secondly, how does webdriverio compare to selenium-webdriverjs? selenium-webdriverjs looks very java-y to me, on the other hand I can't quite fiigure out how webdriverio (yours) does async stuff. 

I basically need help choosing one or the other. Thanks!

### PR DESCRIPTION
**Pull request by dandv**
Hopefully this will drive more adoption now that we've [surpassed 1000 GitHub stars](https://twitter.com/webdriverio/status/622819884452478976)!

Fixes #138.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-![WebdriverIO](http://www.christian-bromann.com/wdio.png)
+we![WebdriverIO](http://www.christian-bromann.com/wdio.png)
```
Comment (christian-bromann) [inline README.md]: is this intentional?

File: README.md [hunk 2]
```diff
@@ -29,6 +64,15 @@ or if you want to use the wdio test runner
 npm install -g webdriverio
 ```
 
+## Getting started
+
+Simply run `wdio config` and the configuration helper wizard will get you set up:
+
+![wdio wizard screen](https://cloud.githubusercontent.com/assets/33569/9005399/b876d33e-3732-11e5-98bc-d47ef030e595.png)
```
Comment (christian-bromann) [inline README.md]: could we use this image instead? http://webdriver.io/images/config-utility.gif

File: README.md [hunk 3]
```diff
@@ -10,14 +10,49 @@ WebdriverIO
...
+  .elementById('q')
+  .sendKeys('webdriver')
+  .elementById('btnG')
+  .click()
+```
+
+For more details on the comparison between WebdriverIO, selenium-webdriverjs and WD.js,
+read [this discussion](https://github.com/webdriverio/webdriverio/issues/138).
+
+## History
```
Comment (christian-bromann) [inline README.md]: can we replace the history section with the plugin section? I think all these images are more catchy to people

File: README.md (hunk 2 above)
Comment (dandv) [inline README.md]: That animated GIF is way cooler :)

File: README.md (hunk 1 above)
Comment (dandv) [inline README.md]: Not at all. I started typing somewhere else and the cursor must've jumped.

File: README.md [hunk 4]
```diff
@@ -10,25 +10,44 @@ WebdriverIO
 
 #### [Homepage](http://webdriver.io) | [Developer Guide](http://webdriver.io/guide.html) | [API Reference](http://webdriver.io/api.html) | [Contribute](http://webdriver.io/contribute.html)
 
-This library is a webdriver module for Node.js. It makes it possible to write
-super easy selenium tests in your favorite BDD/TDD test framework. It was
-originated by [Camilo Tapia's](https://github.com/camme) initial selenium project
-called WebdriverJS which was the first webdriver project on NPM.
+This library is a [webdriver](http://google-opensource.blogspot.com/2009/05/introducing-webdriver.html)
+(browser automation) module for Node.JS. It makes it possible to write
```
Comment (christian-bromann) [inline README.md]: One last thing: could we link to the W3 draft instead? https://w3c.github.io/webdriver/webdriver-spec.html

**Comment by dandv** [comment]
A [comparison with Nightwatch](https://github.com/webdriverio/webdrivercss/issues/14#issuecomment-126872078) would also be interesting, but maybe after this PR is approved.

**Comment by dandv** [comment]
Added a [StackOverflow answer](http://stackoverflow.com/questions/27174083/difference-between-webdriverjs-and-webdriverio/31783517#31783517) based on this conversation.

**Comment by christian-bromann** [comment]
:+1: @dandv

**Pull request merged**